[33mWARNING [0m urllib3.connectionpool:connectionpool.py:780 Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', ConnectionResetError(54, 'Connection reset by peer'))': /session/2bfe4af368a27bff8d1411ccdd1bebc1/screenshot
[33mWARNING [0m urllib3.connectionpool:connectionpool.py:780 Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7ffab32e5940>: Failed to establish a new connection: [Errno 61] Connection refused')': /session/2bfe4af368a27bff8d1411ccdd1bebc1/screenshot
[33mWARNING [0m urllib3.connectionpool:connectionpool.py:780 Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7ffab32e5d00>: Failed to establish a new connection: [Errno 61] Connection refused')': /session/2bfe4af368a27bff8d1411ccdd1bebc1/screenshot
[33mWARNING [0m urllib3.connectionpool:connectionpool.py:780 Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7ffab7881be0>: Failed to establish a new connection: [Errno 61] Connection refused')': /session/2bfe4af368a27bff8d1411ccdd1bebc1/window
[33mWARNING [0m urllib3.connectionpool:connectionpool.py:780 Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7ffab7881a30>: Failed to establish a new connection: [Errno 61] Connection refused')': /session/2bfe4af368a27bff8d1411ccdd1bebc1/window
[33mWARNING [0m urllib3.connectionpool:connectionpool.py:780 Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7ffab7881850>: Failed to establish a new connection: [Errno 61] Connection refused')': /session/2bfe4af368a27bff8d1411ccdd1bebc1/window